app/models.py

```diff
@@ -124,7 +125,7 @@ class User(db.Model):
     )
 
     # either email auth or a mobile number must be provided
-    CheckConstraint("auth_type = 'email_auth' or mobile_number is not null")
+    CheckConstraint("auth_type in ('email_auth', 'webauthn_auth') or mobile_number is not null")
```

Comment:
I think this check constraint also needs updating in the database, so will need an update in the migration.

Reply:
Ahh, good point 👍🏼

Reply:
Added now :3